docs/eureka/devnet-testing-guide.mdx

```diff
@@ -128,6 +129,76 @@ Example Output:
 ETH: 0.092298623946995983
 ```
 
+### Check Balance of Cosmos Account
+
+To check the balance on Cosmos, enter a Cosmos `cosmos1` address into the `[address]` flag.
+
+```bash
+#Usage:
+go run ./ balance [address] [optional-denom-or-erc20-address] [flags]
+```
+
+Example:
+
+```bash
+go run ./ balance cosmos1u5d4hk8294fs9pq556jxmlju2ceh4jmurcpfv7
+```
+
+Output:
+
+```bash
+IBC Denom: ibc/2351096B1729B2C64AED9F6AFD4A4BC28EB56F624881556947A8C48EDB9ED444
+transfer/08-wasm-0/0xa4ff49eb6e2ea77d7d8091f1501385078642603f: 1
+```
+
+You can make an equivalent query using `gaiad` with:
+
+```bash
+gaiad query bank balances cosmos1u5d4hk8294fs9pq556jxmlju2ceh4jmurcpfv7 --chain-id=highway-dev-1 --node=https://eureka-devnet-node-01-rpc.dev.skip.build:443 --output json
+```
+
+### Transfer Tokens from Ethereum to Cosmos (Back Again!)
```

Comment:
I think this should exist above the balance queries, and balance queries last. To keep the one direction and the other commands together for better organization

Reply:
Yeah I'm p indifferent on keeping the balance command instructions for cosmos or not, only main thing for me here is that the transfer commands should all be groped together